pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.blue]
+line-length = 80
```

Comment:
If this isn't going to be 79 as recommended by https://peps.python.org/pep-0008/#maximum-line-length then I propose 99 as per "it is okay to increase the line length limit up to 99 characters" also in PEP-8 :)

Reply:
WTH?! I **specifically** set that to 90... grumble